region/lowernorfair/east.json

```diff
@@ -1192,7 +1191,41 @@
           "to": [
             {
               "id": 4,
-              "requires": [ "canUsePowerBombs" ],
+              "requires": [ 
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},                
+                "canUsePowerBombs",
+                {"or": [
```

Comment:
I get the feeling this `or` closed too early and the walljump and heatframes should've been in an `and`?

Reply:
There's an additional 200 heatframe cost for not being able to walljump up there. The dev will have to add the different heatframes up into one.

Reply:
Oh! Makes sense.